platform/mwaa/main.tf

```diff
@@ -7,6 +7,7 @@ locals {
   default_airflow_configuration_options = {
     "logging.logging_level"      = "INFO"
     "webserver.dag_default_view" = "graph"
+    "webserver.instance_name"    = "${var.prefix} DAGs"
```

Comment:
We can definitely remove this and set it on the veda-data-airflow level. But, is this something we want to standardize here?

Reply:
We should avoid defining a default value for a variable in the TF module at [this line](https://github.com/NASA-IMPACT/mwaa_tf_module/blob/issue-161/airflow-configuration-options/platform/mwaa/variables.tf#L4) because it will always use that default.

Reply:
[74db4a9](https://github.com/NASA-IMPACT/mwaa_tf_module/pull/13/commits/74db4a9c1284affbc7eeac8f010f8f78fa9180b6)